src/components/textarea/index.tsx

```diff
@@ -18,6 +18,7 @@ interface Props {
   onChange?: ChangeEventHandler<HTMLTextAreaElement>;
   rows?: number;
   cols?: number;
+  textStyle?: string;
```

Comment:
Do we want to pass all the typography tokens or the fontStyle in the texarea are restricted to some styles?

Reply:
I don't think we need heading, button and some other textStyle tokens. Would you mind doble checking with the Designer? Thank you 🌸

Reply:
I do not know if there are some restrictions, based on a talk with designer usually there will be `body` and `body-small`. Maybe it makes sense to specify just those two

Reply:
I will pass only the textStyles for its future use/case